src/containers/Header/LanguagePicker/LanguagePicker.tsx

```diff
@@ -15,21 +23,19 @@ export const LanguagePicker = () => {
   return (
     <div className="language-picker">
       <Dropdown
-        title={t(`language`, { context: currentLanguage, defaultValue: '' })}
+        title={lang}
         className="dropdown-right"
       >
         {supportedLanguages
           .filter((language) => language !== currentLanguage)
           .map((language) => (
+
             <DropdownItem
               key={language}
               className={`language-picker-${language}`}
               handler={handleLanguageClick(language)}
             >
-              {t(`language`, {
-                context: language,
-                defaultValue: '',
-              })}
+             {language=='en-US'?"English":language=='ja-JP'?"Japaneese":"Korean"}
```

Comment:
Same here.  This will also make it so less files need to be updated when adding a language.